### PR TITLE
Added Java Sol to cses1634 minimum coins

### PR DESCRIPTION
**Pull request by TruaShamu**

### CODE REVIEW

File: solutions/CSES Minimum Coins.mdx [hunk 1]
```diff
@@ -85,3 +91,65 @@ int main(){
...
+		/* Initialize DP. */
+		int[] dp = new int[X + 1];
+		for (int i = 0; i <= X; i++) {
+			dp[i] = MAX;
+		}
+		dp[0] = 0;
+
+		/* Loop through all coins and all weights
+		If the state curWeight - coin[i] is possible
+		DP[curWeight] = min(DP[curWeight], DP[curWeight - coin[i] + 1). */
```
Comment (dongliu0426) [inline solutions/CSES Minimum Coins.mdx:132]:
```suggestion
		DP[curWeight] = min(DP[curWeight], DP[curWeight - coin[i]] + 1). */
```

**Review by dongliu0426 — CHANGES_REQUESTED** [review]
also update the author list

**Comment by TruaShamu** [comment]
done :)

File: solutions/CSES Minimum Coins.mdx [hunk 2]
```diff
@@ -85,3 +91,65 @@ int main(){
...
 }
 ```
+
+</CPPSection>
+
+
+<JavaSection>
+
+```java
+
```
Comment (dongliu0426) [inline solutions/CSES Minimum Coins.mdx:101]:
```suggestion
```

File: solutions/CSES Minimum Coins.mdx [hunk 3]
```diff
@@ -40,6 +40,12 @@ you used, and print $-1$ if so (since it's impossible to achieve a sum of $x$).
 
 </Warning>
 
+<LanguageSection>
+
+<CPPSection>
+
+## Implementation
```
Comment (dongliu0426) [inline solutions/CSES Minimum Coins.mdx:47]: move `## Implementation` outside `<LanguageSection>` and add `**Time Complexity**: \mathcal O(N\cdot X)` below it

**Review by dongliu0426 — CHANGES_REQUESTED** [review]

**Comment by TruaShamu** [comment]
Sorry. It should be fixed now.

File: solutions/CSES Minimum Coins.mdx [hunk 4]
```diff
@@ -40,6 +40,14 @@ you used, and print $-1$ if so (since it's impossible to achieve a sum of $x$).
 
 </Warning>
 
+## Implementation
+
+**Time Complexity**: \mathcal O(N\cdot X)
```
Comment (dongliu0426) [inline solutions/CSES Minimum Coins.mdx:45]:
```suggestion
**Time Complexity**: $\mathcal O(N\cdot X)$
```
oops

**Review by dongliu0426 — APPROVED** [review]

**Comment by TruaShamu** [comment]
I'm sorry, I'm dumb... but what does this mean?

UI Tests: Unreviewed
1 change must be accepted as baseline to pass UI tests

**Comment by maggie-j-liu** [comment]
I think those are just for storybook, so you don't have to worry about them. The important checks are the pull request tests and the deployment.

**Comment by TruaShamu** [comment]
Okay, thanks!

**Comment by TruaShamu** [comment]
So, if it's passed all checks and has been reviewed, then can the pull request be merged please?

**Pull request merged**